crates/provider/src/kubernetes/namespace.rs

```diff
@@ -419,7 +421,7 @@ where
     }
 
     async fn generate_files(&self, options: GenerateFilesOptions) -> Result<(), ProviderError> {
-        debug!("options {:#?}", options);
+        debug!("generate files options {options:#?}");
```

Comment:
We should remove this `pretty` debugs, since the multiline output isn't optimal.